src/main/java/seedu/address/logic/commands/ConvertCommand.java

```diff
@@ -47,7 +47,7 @@
     //@@author Sheikh-Umar
     public static final String MESSAGE_DUPLICATE_PERSON = "This Lead/Contact already exists in the CRM Book.";
     //@@author
```

Comment:
you can just replace this with //@@author WoodyLau instead of //@@author so you don't need the next line